### ISSUE TITLE
Media indicator SVG stays dark on high contrast mode

### ISSUE THREAD

**Issue by j-pendlebury** [BBC-archive/psammead#786]
**Describe the bug**
When using the high contrast mode on PC, and with colour preferences changed in Firefox, the media indicator SVG icon stays dark and it's hard to see.

![image](https://user-images.githubusercontent.com/13763899/60719494-c639f380-9f1f-11e9-9f04-715cf938be14.png)

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to www.stage.bbc.com/igbo
2. Go to Firefox preferences
3. Look for the Language and Appearance section
![image](https://user-images.githubusercontent.com/13763899/60719546-eec1ed80-9f1f-11e9-8cce-d5ea21bab741.png)
4. Go to Colours
5. Set override to Always
6. Make sure "Use system colours" is **not** ticked
7. Set colours to a black/dark background and text that can be read (something bright)
![image](https://user-images.githubusercontent.com/13763899/60719654-4a8c7680-9f20-11e9-8f11-8bd5e5005268.png)
8. Go back to the Igbo front page - icons are now hard to see
9. Also look at with a custom high contrast mode on PC

**Expected behaviour**
Icon should be visible regardless of what colours you have selected

**Screenshots**
Current:
![image](https://user-images.githubusercontent.com/13763899/60719840-b66edf00-9f20-11e9-80c5-2eafeb458202.png)

Expected:
![image](https://user-images.githubusercontent.com/13763899/60719846-ba9afc80-9f20-11e9-81ab-8c53c978300e.png)
(although should probably talk to Catharine and UX)

**Comment by jblaut** [comment]
Issue created from an a11y swarm here: https://github.com/bbc/psammead/issues/560

**Comment by j-pendlebury** [comment]
Will most likely require changes similar to https://github.com/bbc/psammead/pull/704 and https://github.com/bbc/psammead/pull/787

**Comment by j-pendlebury** [comment]
Check with UX that they're happy with the solution
